apps/base/utils.py

```diff
@@ -103,7 +103,7 @@ def send_email(
     """
     try:
         sg = sendgrid.SendGridAPIClient(
-            apikey=os.environ.get("SENDGRID_API_KEY")
+            apikey=(os.environ.get("SENDGRID_API_KEY"))
```

Comment:
apikey now expects a tuple?

Reply:
no no actually it doesn't matter even if we don't put those parenthesis, if you want then I can remove them